packages/hint-webpack-config/src/config-exists.ts

```diff
@@ -25,7 +25,11 @@ export default class WebpackConfigConfigExists implements IHint {
         const notFound = () => {
             debug(`'parse::error::webpack-config::not-found' received`);
 
-            context.report('', getMessage('configExists', context.language));
+            context.report(
```

Comment:
I've been treating `off` as deletions of tests unless the severity can change for another factor. I think we want to delete this tests and just make the other ones conditional to receive the configuration.

Reply:
this hint only check for that, do we want to delete the whole hint?

Reply:
Honestly, I'm okay with deleting the whole hint. I don't think this one is adding value.

Reply:
ok, I'm going to delete it